cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
- Separate this line to a single function name it "setupViewStyle" to meet the Single Responsibility Principle in the SOLID.
- Move the color object to the AppColor Class for future reuse and also for future changes so you could change from one place.